### ISSUE TITLE
put back act test aria-props-permitted

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#3132]
We had to remove the test so we could release 4.3.3. We need to put it back when we do a 4.4 release which will include the [appropriate fix](https://github.com/dequelabs/axe-core/pull/3102).

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base, `aria-allowed-attribute` rule is now passing for attribute `aria-busy` and `aria-owns` and failing with earlier attribute `aria-selected` and `audio`
Script:
```
<div id="target" tabindex="1" aria-selected="true" aria-checked="true"></div>
      <div id="target" tabindex="1" aria-busy="true" aria-owns="foo"></div>
      <div id="target" tabindex="1" aria-selected="true" aria-owns="foo"></div>
      <audio
  src="/test/assets/moon-speech.mp3"
  controls
  aria-orientation="horizontal"
  id="failCase"
></audio>
```
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/47409579/187334277-8a6890da-897d-44ac-ae67-019a54a61bac.png">
